insuree/gql_mutations.py

```diff
@@ -324,6 +336,99 @@ def async_mutate(cls, user, **data):
             errors = errors[0]['list']
         return errors
 
+class MoveFamilyToParentMutation(OpenIMISMutation):
+    """
+    Moves a family to a parent one
+    """
+    _mutation_module = "insuree"
+    _mutation_class = "MoveFamilyToParentMutation"
+
+    class Input(OpenIMISMutation.Input):
+        family_uuid = graphene.String(required=True)
+        family_uuids = graphene.List(graphene.String, required=True)
+        cancel_policies = graphene.Boolean(default_value=False)
+
+    @classmethod
+    def async_mutate(cls, user, **data):
+        errors = []
+        for child_family_uuid in data["family_uuids"]:
+            if child_family_uuid == data["family_uuid"]:
+                errors.append({
+                    'title': child_family_uuid,
+                    'list': [{'message': _(
+                        "Impossible d'assigner la famille %(id)s à elle-même") % {'id': child_family_uuid}}]
```

Comment:
Make the error message in EN and add the translation in Lokalise. Do the same for the other errors.